src/m365/spo/commands/group/group-member-remove.ts

```diff
@@ -59,7 +68,13 @@ class SpoGroupMemberRemoveCommand extends SpoCommand {
         option: '--groupName [groupName]'
       },
       {
-        option: '--userName <userName>'
+        option: '--userName [userName]'
+      },
+      {
+        option: '--email [email]'
```

Comment:
Could you add some validation for the email field as well? (using `validation.isValidUserPrincipalName`)

And while you're add it, could you add the same validation check for the userName field as well. I think someone forgot that 😉